### ISSUE TITLE
Training Strategy

### ISSUE THREAD

**Issue by BurakSevsay** [chris1004336379/360DVO#6]
Hi,

Thank you for your great work. I really enjoyed reading the paper and experimenting with the model.

In the paper, it is stated that 360DVO is trained on TartanAirV2 using the same training strategy as DPVO. I have been trying to train 360DVO from scratch on TartanAirV2 using the original DPVO training script. I tested several hyperparameter settings, but the training either suffers from exploding gradients or fails to learn effectively.

Are you planning to release the training script soon? Alternatively, could you please clarify the main differences, if any, between your 360DVO training setup and the original DPVO training strategy?

Thank you for your time and help.
